packages/x6-plugin-export/src/index.ts

```diff
@@ -49,6 +49,16 @@ export class Export extends Basecoat<Export.EventArgs> implements Graph.Plugin {
   toSVG(callback: Export.ToSVGCallback, options: Export.ToSVGOptions = {}) {
     this.notify('before:export', options)
 
+    // to keep pace with it's doc description witch, the default value should be true.
+    // without Object.hasOwn method cause by ts config target.
+    // without instance.hasOwnProperty method cause by ts rule.
+    // the condition will be false if these properties have been set undefined in the target,
+    // but will be true if these properties are not in the target, cause the doc.
+    !Object.prototype.hasOwnProperty.call(options, 'copyStyle') &&
```

Comment:
Because the following judgment is `options.copyStyles !== false`, so if it is not passed, it is also executed in accordance with true.

Reply:
In fact, we just have a difference about what is the true . It will execute correctly under the strict-Boolean-type. The `options.copyStyles !== false` is a strict un-equality which likes ===.But, the serializeImages condition in line 209 is not strict which likes ==.So, In my option, we should keep the same style about the serializeImages and copyStyle condition what are both the Boolean value through the docs on the one hand, on the other hand, It should excute false statement when the value is either `''`, or ` `` `, or `null`,or `undefined`,or `NAN`. All above these just beacuse JS is a fuck weak type language😓.

Reply:
👍